src/components/Carousel.test.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Carousel from './Carousel';
+
+describe('<Carousel/>', () => {
+  let wrapper, props;
+
+  beforeEach(() => {
+    props = {
+      images: [
+        {
+          id: '123yvu1231bv23',
+          url: 'https://placeholder.com/480x200.png',
```

Comment:
may i recommend [Fill Murray](https://www.fillmurray.com/)

Reply:
😹

Reply:
🤣 I had no idea this existed. I'll have to keep that in mind for next placeholders.